dev/WebView2/WebView2.cpp

```diff
@@ -833,24 +833,28 @@ void WebView2::CreateMissingAnaheimWarning()
     Content(warning);
 }
 
-// We could have a child TextBlock (see CreateMissingAnaheimWarning), make sure it is visited by Measure pass.
+// We could have a child Grid (see AddChildPanel) or a child TextBlock (see CreateMissingAnaheimWarning).
+// Make sure it is visited by the Measure pass.
 winrt::Size WebView2::MeasureOverride(winrt::Size const& availableSize)
 {
-    //if (auto child = Content().try_as<winrt::FrameworkElement>())
-    //{
-    //    Content().Measure(availableSize);
-    //}
+    if (auto child = Content().try_as<winrt::FrameworkElement>())
+    {
+        child.Measure(availableSize);
```

Comment:
Is this override needed? I'd expect the default implementation inherited from Control to take care of this.